value-processor/src/org/immutables/value/processor/meta/Constitution.java

```diff
@@ -694,6 +694,10 @@ public InnerBuilderDefinition() {
       super(names().namings.typeInnerBuilder);
     }
 
+    protected boolean isCorrectKind(Protoclass p) {
```

Comment:
what `correct` in this context means ? 
Can you find a better name and add javadoc.

Reply:
I can suggest smth like `isApplicableTo(p)` and briefly document on the abstract method level in the base class. Probably, not much better, but at least smth.

Reply:
The original base implementation was looking for Value.Immutable annotation, but that's not revenant for Value.Modifiable only classes.  I've adopted @elucash's suggested name for now.